src/torchaudio/__init__.py

```diff
@@ -40,6 +42,204 @@
 except ImportError:
     pass
 
+# CI cannot currently build with ffmpeg>4, but torchcodec is buggy with ffmpeg4. This hack
+# allows CI to build with ffmpeg4 and works around load/test bugginess.
+if "pytest" in sys.modules:
```

Comment:
We shouldn't do this, a general rule for mocking is that the mocking  should happen within the tests, and only within the tests. We shouldn't modify anything within `src/` for this. If this is not possible, let's try to brainstorm solutions.

(In addition, pytest may be imported in non-test runs, for any reason)